docs2/site/docs/migrations/migration5.md

```diff
@@ -84,6 +84,27 @@ and `GraphQL.NewtonsoftJson` libraries, these serialize and deserialize to JSON
 
 ### 6. `AutoRegisteringObjectGraphType` and `AutoRegisteringInputObjectGraphType` enhancements
 
+These two classes now provide a range of customizable behavior for data models without the
+need for creating individual graph types for each data model. New for v5, fields' names
+and graph types can be customized by applying attributes to the respective property, such
+as is shown in the below example:
+
+```csharp
+// graph type: AutoRegisteringObjectGraphType<Person>
+
+class Person
+{
+    [Name("Id")]
+    [OutputType(typeof(IdGraphType))]
+    public int PersonId { get; set; }
+
+    public string Name { get; set; }
+
+    [GraphQLAuthorize("Administrators")]
```

Comment:
The presence of the prefix causes questions. Unify naming adding `GraphQL` prefix to all attributes?

Reply:
I really wouldn't.  It will become exceedingly obvious why once I continue with future PRs for auto registering output class.  We can change it later if desired.

Reply:
ok